cms/search/serializers.py

```diff
@@ -0,0 +1,89 @@
+from typing import TYPE_CHECKING
```

Comment:
My concern is that this is duplicating a lot of what we already have in the search publisher. Have you looked at reusing/refactoring existing code so we are not constructing the message in multiple places? Ideally, we construct in 1 place, for search and the view. Right now, they are out of sync, and changes would have to be made in both places.